http.go

```diff
@@ -70,6 +70,10 @@ func (s *Server) ServeHTTPS() {
 		log.Fatalf("FATAL: loading tls config (%s, %s) failed - %s", s.Opts.TLSCertFile, s.Opts.TLSKeyFile, err)
 	}
 
+	if len(s.Opts.TLSClientCAFiles) > 0 {
+		config.ClientAuth = tls.RequestClientCert
```

Comment:
including ClientCAs scopes the request ("please send me a client cert signed by X"). omitting it causes client cert prompts in lots of browsers, even if none of the client certs they have available would be acceptable. you should definitely set it to the pool you are going to use to verify.